### PR TITLE
Add modules for Quotes and Pinterest Pins extraction

### PR DESCRIPTION
**Pull request by KomalBhadani27**
- Add quote_extractor
- Add embedded_p_pins_extractor

### CODE REVIEW

File: extraction/extractor.py [hunk 1]
```diff
@@ -85,8 +86,14 @@ def __extract_data_from_html_body(self, node):
     def __validate_and_extract_content(self, node):
         """This function will extract valid content and return it"""
 
-        # This function will be updated in next iteration
-        _ = node
-        _ = self.soup
+        content_extractors = list()
+        content_extractors.append(quote_extractor.QuoteExtractor())
+        content_extractors.append(embedded_p_pins_extractor
+                                  .EPinterestPinExtractor())
```
Comment (detel) [inline extraction/extractor.py:92]: You can make this list global. Also, convert it into tuple.

Comment (KomalBhadani27) [reply extraction/extractor.py]: Updated accordingly! Thanks for the suggestions.

File: extraction/embedded_p_pins_extractor.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+"""This script checks whether DOM has pinterest pins tag or not and
```
[the remark this answers is not in the document]
Comment (KomalBhadani27) [reply extraction/embedded_p_pins_extractor.py:1]: embedded_pinterest_pin_extractor.py or any other suggestions?

[the remark this answers is not in the document]
Comment (detel) [reply extraction/embedded_p_pins_extractor.py:1]: Sounds good.

**Pull request merged**